staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go

```diff
@@ -1534,6 +1534,11 @@ func describePersistentVolumeClaim(pvc *corev1.PersistentVolumeClaim, events *co
 		if pvc.Spec.VolumeMode != nil {
 			w.Write(LEVEL_0, "VolumeMode:\t%v\n", *pvc.Spec.VolumeMode)
 		}
+		if pvc.Spec.DataSource != nil {
+			w.Write(LEVEL_0, "DataSource:\n")
+			w.Write(LEVEL_1, "Name:\t%v\n", pvc.Spec.DataSource.Name)
```

Comment:
I am confused why not print pvc.Spec.DataSource.APIGroup at the same time

Reply:
Yes, I think printing out api group makes sense too. Can you open a PR to add it? Right now, since only cloning is beta and on by default, then this can only be PVC type.

Reply:
> Yes, I think printing out api group makes sense too. Can you open a PR to add it? Right now, since only cloning is beta and on by default, then this can only be PVC type.

Sure, I will open a pr.